source/Camera/Advanced/CaptureHDRPrintNormals/CaptureHDRPrintNormals.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+Capture Zivid point clouds, compute normals and print a subset.
+
+For scenes with high dynamic range we combine multiple acquisitions to get an HDR point cloud.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+namespace
+{
+    void printNormals(int radius,
+                      const Zivid::Array2D<Zivid::NormalXYZ> &normals,
+                      const int numOfRows,
+                      const int numOfCols)
+    {
+        const auto lineSeparator = std::string(50, '-');
+        std::cout << lineSeparator << std::endl;
+        for(int row = (numOfRows / 2 - radius); row < (numOfRows / 2 + radius); row++)
+        {
+            for(int col = (numOfCols / 2 - radius); col < (numOfCols / 2 + radius); col++)
+            {
+                std::cout << "Normals (" << row << "," << col << "): ";
+                std::cout << std::setprecision(1) << std::fixed << normals(row, col) << std::endl;
```

Comment:
I am not sure this works (set precision and fixed) on normals(row, col). Did you test it?